app/build.gradle

```diff
@@ -30,12 +30,18 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
         manifestPlaceholders.isRaptorEnabled = "false"
     }
+
     buildTypes {
         release {
             shrinkResources true
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+        raptor {
```

Comment:
FYI, I just noticed this new build type needs to be whitelisted at https://github.com/mozilla-mobile/fenix/pull/1220/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R450. Please let me know once the name is define, so I can update #1220 accordingly.